docs/apache-airflow-providers-ssh/connections/ssh.rst

```diff
@@ -99,7 +99,7 @@ Extra (optional)
 
     .. code-block:: bash
 
-       python -c 'import re, sys; print(re.sub("\r\n", "\\\\n", sys.stdin.read()))' < /path/to/your/key
```

Comment:
I think it depends on where you are connecting.

I guess the right pattern will be `\r?\n`)